packages/e2e/test/artillery/wallet-restoration/WalletRestoration.ts

```diff
@@ -2,21 +2,22 @@ import { AddressType, GroupedAddress, util } from '@cardano-sdk/key-management';
 import { AddressesModel, WalletVars } from './types';
 import { Cardano } from '@cardano-sdk/core';
 import { FunctionHook } from '../artillery';
-import { Pool, QueryResult } from 'pg';
 import { StubKeyAgent, getEnv, getWallet, walletVariables } from '../../../src';
-import { findAddressesWithRegisteredStakeKey } from './queries';
 import { logger } from '@cardano-sdk/util-dev';
-import { walletReady } from '../../util';
-
-const env = getEnv([
-  ...walletVariables,
-  'DB_SYNC_CONNECTION_STRING',
-  'ARRIVAL_PHASE_DURATION_IN_SECS',
-  'VIRTUAL_USERS_COUNT'
-]);
+import { waitForWalletStateSettle } from '../../util';
 
+const env = getEnv([...walletVariables, 'ARRIVAL_PHASE_DURATION_IN_SECS', 'VIRTUAL_USERS_COUNT']);
 const operationName = 'wallet-restoration';
-const SHORTAGE_OF_WALLETS_FOUND_ERROR_MESSAGE = 'Addresses found from db are less than desired wallets count';
+
+const getNetworkName = ({ networkMagic }: Cardano.ChainId) => {
+  switch (networkMagic) {
+    case Cardano.NetworkMagics.Mainnet:
+      return 'mainnet';
+    case Cardano.NetworkMagics.Preprod:
+      return 'preprod';
+  }
+  throw new Error(`Unsupported network magic ${networkMagic} for the load test`);
```

Comment:
Is there much complexity with retaining the current behaviour as a fall through, so we can run it on our local environment as a way to verify the load test run as we expect it to?

Reply:
Good idea, to use DB query against our local network env, shouldn't be complex. @rhyslbw